apps/site/lib/site/transit_near_me.ex

```diff
@@ -42,31 +42,20 @@ defmodule Site.TransitNearMe do
     schedules_fn: &Schedules.Repo.schedule_for_stop/2
   ]
 
-  @spec build(Address.t(), Keyword.t()) :: t() | error
+  @spec build(Address.t(), Keyword.t()) :: map
```

Comment:
Could we update the defstruct above to not include schedules and keep this as `t()`?